### DISCUSSION TITLE
How can i can we add both forknet and mainnet of same chainID

### DISCUSSION THREAD

**Discussion by Anirudha619** [rainbow-me/rainbowkit#1841] [Q&A]
I want to add forknet and mainnet rpc for same chainId in rainbowKit.
How to Do so?

**Comment by 0xmkh** [comment]
@Anirudha619 You'll get issues if you add same chain id for two chains , since one might override the other. You could add your custom chain via RainbowKit here: https://www.rainbowkit.com/docs/custom-chains

Feel free to use that approach if you can 🙏
